README.md

```diff
@@ -184,14 +189,149 @@ Binaries built by Burrito include a built-in set of commands for performing main
 
 * `./my-binary maintenance uninstall` - Will prompt to uninstall the unpacked payload on the host machine.
 
+## Advanced Build Configuration
+
+#### Build Steps and Phases
+
+Burrito runs the mix release task in three "Phases". Each of these phases contains any number of "Steps" -- a context struct is passes between each step that contains the current state of the build tree and mix release.
```

Comment:
Typo: passes -> passed

Also, the em dash is throwing off the flow of the sentence for me a little bit; how do you feel about this?

> Each of these phases contains a number of "Steps", and a context struct, containing the current state of the build, is passed between each step.